plymouth/qubes-dark/qubes-dark.script

```diff
@@ -6,9 +6,19 @@ logo.sprite = Sprite(logo.image);
 logo.opacity_angle = 0;
 
 fun refresh_callback () {
+    # Reposition everything 50 times per second
```

Comment:
This looks scary ;) But if it helps, I'm happy :)

Reply:
> This looks scary ;) But if it helps, I'm happy :)

I am not 100% sure if it fixes it. Actually the logo is repositioned every 50 seconds at the moment with the existing code before this fix. I am guessing that the plymouth render interleaves the default 50 refresh cycles between the two monitors. But I might be wrong.

Reply:
So it was a failure:

https://openqa.qubes-os.org/tests/106830#step/firstboot/3

Reply:
This one is with the update applied: https://openqa.qubes-os.org/tests/106932#step/startup/4
So, not looking good. When the current run completes, I'll double check if initramfs actually include your change.

Reply:
In the plymouth issue I included a debug log, but also info in which plymouth version it worked before. Maybe that will help...